lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2555,7 +2593,11 @@ def test_calculate_report_csv_already_running(self, report_type, instructor_api_
         with patch(task_api_endpoint) as mock:
             mock.side_effect = AlreadyRunningError()
             response = self.client.get(url, {})
-        already_running_status = "{report_type} report generation task is already in progress. Check the 'Pending Instructor Tasks' table for the status of the task. When completed, the report will be available for download in the table below.".format(report_type=report_type)
+        already_running_status = "An {report_type} report is currently in progress." \
+                                 " To view the status of the report, see the 'Pending Instructor Tasks' section." \
+                                 " When completed, the report will be available for download in the table below." \
+                                 " You will be able to download the" \
+                                 " report when it is complete.".format(report_type=report_type)
```

Comment:
Change to:

"An {report_type} report is currently in progress." \
+                                 " To view the status of the report, see Pending Instructor Tasks below." \
+                                 " You will be able to download the" \
+                                 " report when it is complete."

(Delete "When completed..." line; change "see the 'Pending Instructor Tasks' section)